src/xrpld/app/tx/detail/Payment.cpp

```diff
@@ -342,37 +355,16 @@ Payment::doApply()
         view().update(sleDst);
     }
 
-    // Determine whether the destination requires deposit authorization.
```

Comment:
I can be overruled but I think it's pretty hard to compare pre and post amendment code here. Should not delete this code. Add amendment check under `bRipple` expression. If enabled then execute the new code with `DepositPreauth::applyCheckCredentials`, otherwise execute the old code.

Reply:
fixed